src/gen/engine/src/main/java/com/paxful/sdk/PxsdkGenerator.java

```diff
@@ -0,0 +1,66 @@
+package com.paxful.sdk;
+
+import io.swagger.codegen.v3.*;
+import io.swagger.codegen.v3.generators.DefaultCodegenConfig;
+import io.swagger.codegen.v3.generators.typescript.TypeScriptFetchClientCodegen;
+import io.swagger.v3.oas.models.Operation;
+import io.swagger.v3.oas.models.media.Schema;
+import io.swagger.v3.oas.models.parameters.Parameter;
+
+import java.util.*;
+import java.io.File;
+
+public class PxsdkGenerator extends TypeScriptFetchClientCodegen {
+
+  // source folder where to write the files
+  protected String sourceFolder = "src";
+  protected String apiVersion = "1.0.0";
+
+  /**
+   * Configures the type of generator.
```

Comment:
If all methods are overrides of methods from parent class, then their docs can simply include "inheritDoc"

Reply:
Just removed the comment block - no reason to keep it all.